bot/exts/evergreen/wikipedia.py

```diff
@@ -103,6 +101,10 @@ def check(message: Message) -> bool:
             except IndexError:
                 await ctx.send(f"Sorry, please give an integer between `1` to `{titles_len}`, {error_msg}")
 
+            except Exception as e:
+                log.info(e)
+                break
+
```

Comment:
Maybe you could add a message on `log.info`? Something like `log.info(f"Caught exception {e}, breaking out of retry loop")`